### ISSUE TITLE
Feature Suggestion: Patterns: Divider output triggers on first clock.

### ISSUE THREAD

**Issue by Chaircrusher** [jeremywen/JW-Modules#49]
As discussed on the VCVRack forum.

Add an option -- context menu, or even better, per-divisor toggle -- to have the Nth divisor put out a trigger on the first clock pulse after reset.

In other words: consider the /4 output. It will generate this stream of pulses
. . . | . . . | . . . | . . . | . . . | . . . | . . . | . . . | . . . | . . . | . . . | . . . | . . . | . . . | . . . | 
With 'start from 1st pulse mode:'
| . . . | . . . | . . . | . . . | . . . | . . . | . . . | . . . | . . . | . . . | . . . | . . . | . . . | . . . | . . .

**Comment by DaveBenham** [comment]
Yes please! An option for all divisions to fire on the 1 after a reset would be extremely helpful (and musical)

This would be true for 1Pattern as well.
